src/Symfony/Component/HttpFoundation/Exception/ProblemJsonResponseException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Exception;
+
+/**
+ * @author Abdellah Ramadan <ramadanabdel24@gmail.com>
+ */
+class ProblemJsonResponseException extends UnexpectedValueException
```

Comment:
The scope of this exception looks too broad, it's a concrete one yet no one can guess where it's meant to be thrown by looking at its class only. As this feature's scope is not huge and I see no use case for extending such exception, I don't think it's worth adding it.